study-builder/studies/basil/i18n/en.conf

```diff
@@ -11,6 +11,151 @@
     "name": "Picklist Autocomplete",
     "title": "Title: Picklist Autocomplete feature",
   },
+  "options": {
+    "yes": "Yes",
+    "no": "No"
+  },
+  "matrix": {
+    "name": "Matrix Questions",
+    "title": "Title: Matrix Questions",
+    "sections": {
+      "first": {
+        "prompt": "MATRIX QUESTION: SINGLE MODE"
+      },
+      "second": {
+        "prompt": "MATRIX QUESTION: SINGLE MODE with GROUPS"
+      },
+      "third": {
+        "prompt": "MATRIX QUESTION: MULTIPLE MODE"
+      },
+      "fourth": {
+        "prompt": "MATRIX QUESTION: MULTIPLE MODE with GROUPS"
+      }
+    },
+    "required": {
+      "hint": "Required question!"
+    },
+    "matrix_single_mode": {
+      "prompt": "Matrix Question: Single Mode"
+      "options": {
+        "never": {
+          "prompt": "Never"
+        },
+        "once": {
+          "prompt": "Once a week"
+        },
+        "twice": {
+          "prompt": "Twice a week"
+        }
+      },
+      "rows": {
+        "meat": {
+          "prompt": "How often do you eat meat?"
+        },
+        "fruit": {
+          "prompt": "How often do you eat fruits?"
+        }
+      }
+    },
+    "matrix_single_mode_w_groups": {
+      "prompt": "Matrix Question: Single Mode w/ Groups"
+      "groups": {
+        "first": {
+          "prompt": "EXAMPLE OF FIRST GROUP"
+        },
+        "second": {
+          "prompt": "EXAMPLE OF SECOND GROUP"
+        }
+      },
+      "options": {
+        "never": {
+          "prompt": "Never"
+        },
+        "once": {
+          "prompt": "Once a week"
+        },
+        "twice": {
+          "prompt": "Twice a week"
+        }
+      },
+      "rows": {
+        "meat": {
+          "prompt": "How often do you eat meat?"
+        },
+        "fruit": {
+          "prompt": "How often do you eat fruits?"
+        }
+      }
+    },
+    "matrix_multi_mode": {
+      "prompt": "Matrix Question: Multiple Mode"
+      "options": {
+        "never": {
+          "prompt": "Never"
+        },
+        "once": {
+          "prompt": "Once a week"
+        },
+        "twice": {
+          "prompt": "Twice a week"
+        }
+      },
+      "rows": {
+        "meat": {
+          "prompt": "How often do you eat meat?"
+        },
+        "fruit": {
+          "prompt": "How often do you eat fruits?"
+        }
+      }
+    },
+    "matrix_multi_mode_w_groups": {
+      "prompt": "Matrix Question: Multiple Mode w/ Groups"
+      "groups": {
+        "first": {
+          "prompt": "HOW OFTEN"
```

Comment:
Is using `prompt` accurate here? I don't see any references to `prompt` when working with the `MatrixGroupDef`, `JdbiMatrixGroup`, or `MatrixGroup` types.

Reply:
It's just a variable name. The actual config is here:
<img width="1007" alt="image" src="https://user-images.githubusercontent.com/71830706/144287082-dd6f2f3a-e9cf-437a-a6a5-9feef06634b3.png">